common/dynasties/2000_orc.txt

```diff
@@ -923,4 +923,5 @@
 2754 = {name = "Adgor'gar" culture = frostwolf}
```

Comment:
The character that arrived to C'Thun isn't a follower of C'Thun.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79171089-386e2b80-7e02-11ea-938f-a79d39ef68f9.png)

</details>

Reply:
> The character that arrived to C'Thun isn't a follower of C'Thun.

Well, I can't confirm it can't happen now.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79262567-97837d00-7ea2-11ea-911b-1d699602f638.png)
![image](https://user-images.githubusercontent.com/46384992/79262584-9d795e00-7ea2-11ea-83b8-b4aa2c281440.png)
![image](https://user-images.githubusercontent.com/46384992/79262647-b124c480-7ea2-11ea-8117-4d68a372cb41.png)

</details>